inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/SenderGroup.java

```diff
@@ -110,6 +110,7 @@ public SenderResult send(ByteBuf dataBuf) {
             }
             if (channel == null) {
                 logger.error("can not get a channel");
+                dataBuf.release();
```

Comment:
here need release too
if (channels.size() <= 0) {
                logger.error("channels is empty");
                return new SenderResult("channels is empty", 0, false);
            }